src/main/java/cubicchunks/world/IOpacityIndex.java

```diff
@@ -47,6 +47,16 @@
 	void onOpacityChange(int localX, int blockY, int localZ, int opacity);
 
 	/**
+	 * Returns true if the block at the given position is occluded by a known non-opaque block further up.
+	 *
+	 * @param localX local block x-coordinate (0..15)
+	 * @param blockY global block y-coordinate
+	 * @param localZ local block z-coordinate (0..15)
+	 * @return true if there exists a known non-opaque block above the given y-coordinate
+	 */
+	boolean isOccluded(int localX, int blockY, int localZ);
```

Comment:
Why add yet another method to interface if it  can be already done using everything that already is there with no performance difference?